demo/demo.sh

```diff
@@ -25,8 +25,8 @@ run () {
         echo "$title '$t'; $(cprofile ${BASE_PATH}/$@) 2>&1 | tee ${BASE_PATH}/$t.log; sleep infinity"
 }
 
-run_router="$(run router soma-subscriptions-producer)"
```

Comment:
> Please use `eosdatadistribution/tools/ostree_store.py` or so. Also, use `python -m eosdatadistribution.foo` instead of running scripts directly so that the search path is set up correctly.

In this case we can’t use `-m`, since it’s already being used: `demo.sh` actually runs `python -m cProfile -o eosdatadistribution/foo.prof eosdatadistribution/foo.py`, and cprofile appears to only take Python filenames as arguments.